salt/modules/zypper.py

```diff
@@ -649,10 +649,10 @@ def version_cmp(ver1, ver2, ignore_epoch=False):
     return __salt__['lowpkg.version_cmp'](ver1, ver2, ignore_epoch=ignore_epoch)
 
 
-def list_pkgs(versions_as_list=False, **kwargs):
+def list_pkgs(versions_as_list=False, attr=None, **kwargs):
```

Comment:
I agree with @isbm and @moio here but I'll give @damon-atkins a chance to reply before we move forward with this. Thanks.